### ISSUE TITLE
WCAG Failures for SC 1.4.13: Content on Hover or Focus (for the tooltip)

### ISSUE THREAD

**Issue by juliemoynatPro** [equalizedigital/accessibility-new-window-warnings#11]
Hi,

I've tested the WordPress plugin and it looks really cool but I can't use it for my projects because it's unfortunately not compliant with the [WCAG Success criterion 1.4.13 "Content on Hover or Focus"](https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html).

Actually, the tooltip has these two failures (that you can find at the end of the page I've linked above):
- [F95: Failure of Success Criterion 1.4.13 due to content shown on hover not being hoverable](https://www.w3.org/WAI/WCAG21/Techniques/failures/F95)
- Failure to make content dismissable without moving pointer hover or keyboard focus

For the second point, it could be solved by introducing the ability to dismiss the tooltip with the Escape key.

I hope this can help. Thank you for your work!

**Comment by amberhinds** [comment]
Thanks @juliemoynatPro, we'll work on these enhancements.
